public/js/helpers/chartsHelpers.js

```diff
@@ -33,4 +33,18 @@ const percentageDataSet = (dataSet, yTotalsList) => {
 // Converts all datasets y values in a series to their percentage value
 const formattedSeries = (series, yTotalsList) => series.map((singleSeries) => { return { data: percentageDataSet(singleSeries.data, yTotalsList)};});
 
-export { createPartialsList, formattedSeries, createYTotalsList };
\ No newline at end of file
+// Sort by more recent date
+const compareDates = (a, b) => a.x < b.x ? -1 : 1;
+
+// Fill in empty datapoints with the corresponding missing date and a value of 0 for the content produced on that day
+const fillMissingDates = (startDate, endDate, data) => {
+    const now = startDate.startOf('day').clone();
+    while (now.isBefore(endDate) || now.isSame(endDate)) {
```

Comment:
minor: could use [moment-range](https://www.npmjs.com/package/moment-range) to avoid mutating `now`, it does mean adding a library though...